### PR TITLE
Project Coverage Metrics

### PR DESCRIPTION
**Pull request by yanqiuzhang**
#### Description
As a development team,
In order to see how much of our code is covered by tests,
We would like to measure test coverage using a remote service.

Changes proposed in this pull request:

* Add and configure coveralls

#### What I have learned working on this feature: 
To add and configure coveralls

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -10,6 +10,8 @@
   exit 1
 end
 
+require 'coveralls'
```
Comment (vick3d) [inline spec/rails_helper.rb]: put this on top of the file

File: features/support/env.rb [hunk 2]
```diff
@@ -1,4 +1,7 @@
 require 'cucumber/rails'
+require 'coveralls'
+Coveralls.wear_merged!('rails')
```
Comment (vick3d) [inline features/support/env.rb]: add a line between 2 and 3

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -10,6 +10,8 @@
...
+Coveralls.wear_merged!('rails')
 RSpec.configure do |config|
```
Comment (vick3d) [inline spec/rails_helper.rb]: add a line between 14 and 15

File: spec/rails_helper.rb (hunk 1 above)
Comment (yanqiuzhang) [reply spec/rails_helper.rb]: done

File: features/support/env.rb (hunk 2 above)
Comment (yanqiuzhang) [reply features/support/env.rb]: done

**Pull request merged**